api/main_endpoints/routes/User.js

```diff
@@ -397,7 +397,6 @@ router.post('/getUserById', async (req, res) => {
     }
 
     const { password, ...omittedPassword } = result._doc;
-
```

Comment:
undo the change in this file, i.e. add the blank line back

Reply:
done

Reply:
by undoing i would push a commit, and then check in the "files changed" page to make sure `api/main_endpoints/routes/User.js` is not listed in the changes

Reply:
done for good